### PR TITLE
login and signup pages added

### PR DESCRIPTION
**Pull request by Sandesh9834**
image

image

### CODE REVIEW

File: Styles/signup.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    .login-container {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      background-color: var(--white-color);
+      transform: translate(-50%, -50%);
+      width: 410px;
+      height: 450px;
```
Comment (kushalsourav) [inline Styles/signup.css:29]: use rems instead of px

**Review by kushalsourav — APPROVED** [review]
Everything looks good for me , i think you have missed using rems in lot of places check it out

File: Styles/login.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  line-height: 1.3rem;
+}
+
+.login-container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  background-color: var(--white-color);
+  transform: translate(-50%, -50%);
+  width: 410px;
```
Comment (tarunsankhla) [inline Styles/login.css:16]: Try using rem

File: Styles/signup.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.form-link-group {
+      padding: 10px;
+      display: flex;
+      align-items: center;
+    }
+    
+    .form h2 {
```
Comment (tarunsankhla) [inline Styles/signup.css:7]: Check the spacing and intendation

File: pages/signup.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+          </button>
+          <button  class="btn-badge badge-lg">
+            <a href="cart.html">
+              <i class="fas fa-shopping-cart fa-2x"></i>
+            </a>
+            <span class="icon-button-badge flex center icon-xl">10</span>
+          </button>
+        </div>
+      </nav>
+    <div class="center">
```
Comment (tarunsankhla) [inline pages/signup.html:35]: Add short comment in your html code get easier for reviewer to review

**Review by tarunsankhla — APPROVED** [review]
Overall the code is good , just minor changes

File: pages/signup.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      <div class="login-container">
+        <div class="login-heading">Sign Up</div>
+        <form action="#">
+            <div class="data">
+                <label class="label-class" for="">Email Address</label>
+                <input type="text" placeholder="Enter Email ID" required />
+              </div>
+              <div class="data">
+                <label class="label-class" for="">Password</label>
```
Comment (HirenChafekar) [inline pages/signup.html:44]: Do not add empty values like for="".
Try to use rem instead of px.
Everything else looks good.

**Review by HirenChafekar — COMMENTED** [review]
Do not add empty values like for="".
Try to use rem instead of px.
Everything else looks good.

**Pull request merged**